arrows/ocv/inpaint.h

```diff
@@ -0,0 +1,55 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#ifndef KWIVER_ARROWS_OCV_INPAINT_
+#define KWIVER_ARROWS_OCV_INPAINT_
+
+#include <arrows/ocv/kwiver_algo_ocv_export.h>
+
+#include <vital/algo/image_filter.h>
+
+namespace kwiver {
+
+namespace arrows {
+
+namespace ocv {
+
+/// OCV image inpainting process
+///
+/// Create a masked RGB image from a RGBA image
+class KWIVER_ALGO_OCV_EXPORT inpaint
+  : public vital::algo::image_filter
+{
+public:
+  PLUGIN_INFO( "ocv_inpainting",
+               "Inpaint a RGBA image using non-zero A values as a mask." )
```

Comment:
```suggestion
               "Inpaint an RGBA image using non-zero alpha values as a mask." )
```
(I could probably go either way on 'A' vs. 'alpha', but "**an** RGBA", please 🙂 .)